tests/models/test_solution_strategy.py

```diff
@@ -443,14 +448,25 @@ def test_check_convergence(
     # ``reference_residual`` is not used by ``check_convergence``. We pass a zero
     # arrray.
     nl_params: dict[str, Any] = {
-        "nl_convergence_tol": 1e-5,
-        "nl_convergence_tol_res": 1e-5,
-        "nl_divergence_tol": 1e4,
+        "nl_convergence_tol": ConvergenceTolerance(
+            tol_increment=1e-5,
+            tol_residual=1e-5,
+            max_increment=1e4,
+            max_residual=1e4,
+        )
     }
-    converged, diverged = check_convergence_test_model.check_convergence(
-        nonlinear_increment, residual, np.zeros(1), nl_params
+    # Standard setup of a convergence absolute convergence criterion - typically
+    # orchestrated by a nonlinear solver.
+    convergence_criterion = AbsoluteConvergenceCriterion()
+    convergence_criterion.set_reference_value(0.0, 0.0)
```

Comment:
Would it make sense for absolute criteria to default to 0, 0 as reference values?